common/source/docs/common-video-landingpage.rst

```diff
@@ -178,6 +178,26 @@ Unboxing video
 
 .. youtube:: DPHKe86SiqI
 
+XBLink 4G LTE Transmission System
+=================================
+.. figure:: https://xbstation.com/assets/images/xblink.png
+   :target:  https://xbstation.com/store/xblink
+   
+Overview
+--------
+XBLink is a Plug and Play hardware for BVLOS Flight based on Real time 4G LTE connective. Perfect fit with Cube.
+
+Features
+--------
+* Control UAV via MissionPlanner and QGroundControl
+* Platform supported: MacOS, WindowsOS, AndroidOS
+* MAVLink compatible
+* Low latency HD video streaming
```

Comment:
Well 300-600ms isn't really low latency

Reply:
```suggestion
* HD video streaming (300-600ms latency)
```

Reply:
if you like I can make that change and merge for you...

Reply:
Thanks @khancyr  for your comments.
Thanks @Hwurzburg , I committed your suggestion.